homeassistant/components/hassio/__init__.py

```diff
@@ -520,7 +520,7 @@ def _async_setup_hardware_integration(_: datetime | None = None) -> None:
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up a config entry."""
     dev_reg = dr.async_get(hass)
-    coordinator = HassioDataUpdateCoordinator(hass, entry, dev_reg)
+    coordinator = HassioDataUpdateCoordinator(hass, dev_reg)
```

Comment:
@joostlek `entry` seems to be required. Would you mind to take another look? I believe this PR can be closed at this point or did you have something else in mind here?
https://github.com/home-assistant/core/blob/bd9070be11e8d9f46efd5c49e24dc9b7d530c27b/homeassistant/components/hassio/data.py#L278-L283